config/webpack.config.js

```diff
@@ -147,6 +147,10 @@ module.exports = env => {
       ),
     },
 
+    resolveLoader: {
+      modules: resolveModules,
```

Comment:
The default is 
```
  modules: [ 'node_modules' ],
  extensions: [ '.js', '.json' ],
  mainFields: [ 'loader', 'main' ]
```
https://webpack-3.cdn.bcebos.com/configuration/resolve/#resolveloader
Should we add `extensions` and `mainFields`?
@sharvit

Reply:
Isn't this *merged* with the defaults?